src/components/poll/poll.js

```diff
@@ -45,7 +45,7 @@ class Poll extends React.Component {
 }
 
 Poll.propTypes = {
-  children: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+  children: PropTypes.node.isRequired,
```

Comment:
required:
we need to move `poll.js` and `scanDownload.js` into there own PR pointed at the `dev` branch

reason:
because we're doing feature branching with `forms` and the related fields as a standalone aspect we have to be diligent in trying to avoid potential merge conflicts, and focused on the feature itself, forms. Both `poll.js` and `scanDownload.js` are  outside of that feature scope and can be updated on the primary development branch.

Reply:
Thank you - makes sense! 

Opened a PR: #154